android-ubuntu-24.04/Containerfile

```diff
@@ -0,0 +1,92 @@
+FROM ubuntu:24.04
+
+RUN \
+	apt-get update && \
+	DEBIAN_FRONTEND="noninteractive" apt-get install -y -q --no-install-recommends \
+		-o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" \
+		bc \
+		bison \
+		bsdmainutils \
+		build-essential \
+		chrpath \
+		cpio \
+		diffstat \
+		file \
+		flex \
+		g++-multilib \
+		gawk \
+		gcc-multilib \
+		git-core \
+		gnupg \
+		iputils-ping \
+		less \
+		libegl1-mesa-dev \
+		libgmp-dev \
+		libmpc-dev \
+		libncurses-dev \
+		libsdl1.2-dev \
+		libssl-dev \
+		locales \
+		lz4 \
+		openssh-client \
+		pylint \
+		python3 \
+		python3-git \
+		python3-jinja2 \
+		python3-pexpect \
+		python3-pip \
+		python-is-python3 \
+		socat \
+		sudo \
+		texinfo \
+		tmux \
+		unzip \
+		vim \
+		wget \
+		xterm \
+		zstd \
+	&& rm -rf /var/lib/apt/lists/* && \
+	locale-gen en_US.UTF-8
+
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
+
+RUN \
+	apt-get update && \
+	apt-get install -y -q --no-install-recommends \
+		-o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" \
+		ca-certificates \
+	&& rm -rf /var/lib/apt/lists/* && \
+	wget -O /usr/local/bin/phyLinux https://download.phytec.de/Software/Linux/Yocto/Tools/phyLinux && \
+	chmod a+x /usr/local/bin/phyLinux && \
+	wget -O /usr/local/bin/repo https://storage.googleapis.com/git-repo-downloads/repo && \
+	chmod a+x /usr/local/bin/repo
+
+# Additional Android packages
+RUN \
+	apt-get update && \
+	DEBIAN_FRONTEND="noninteractive" apt-get install -y -q --no-install-recommends \
+		-o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" \
+		curl \
+		dwarves \
+		fontconfig \
+		git-core \
+		lib32z1-dev \
+		libc6-dev-i386 \
+		libelf-dev \
+		libgl1-mesa-dev \
+		libtinfo6 \
+		libx11-dev \
+		libxml2-utils \
+		rsync \
+		swig \
+		python3-dev \
+		x11proto-core-dev \
+		xsltproc \
+		zip \
+		zlib1g-dev \
+	&& rm -rf /var/lib/apt/lists/* && \
+	locale-gen en_US.UTF-8
+
+# Android requires libncurses5 and libtinfo5 but Ubuntu doesn't provide them anymore
+RUN ln -s /usr/lib/x86_64-linux-gnu/libncurses.so.6 /usr/lib/x86_64-linux-gnu/libncurses.so.5
+RUN ln -s /usr/lib/x86_64-linux-gnu/libtinfo.so.6 /usr/lib/x86_64-linux-gnu/libtinfo.so.5
```

Comment:
@smk-embedded this is a little bit hacky but Android still requires libncurses and libtinfo 5. Do you know a better way to install those older versions? There are no apt packages anymore. I think one solution is to manually download them.